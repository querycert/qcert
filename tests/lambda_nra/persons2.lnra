/* Example returning a newly created record */

Companies.flatmap{ c => c.departments.map{ d => struct(company: c.cname, dept: d) } }

