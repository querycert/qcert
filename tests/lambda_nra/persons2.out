[{"company": "HBO", "dept": "HR"}, {"company": "HBO", "dept": "Research"}, {"company": "CNN", "dept": "M&A"}, {"company": "CNN", "dept": "Corporate"}, {"company": "ATT", "dept": "Mobile"}, {"company": "ATT", "dept": "Cable"}]